FreeRTOS/Test/CMock/smp/multiple_priorities_no_timeslice/covg_multiple_priorities_no_timeslice_utest.c

```diff
@@ -732,177 +733,374 @@ void test_coverage_xTaskGetCurrentTaskHandleCPU_invalid_core_id_lt( void )
     TEST_ASSERT( xTaskHandle == NULL );
 }
 
-/*
-The kernel will be configured as follows:
-    #define configNUMBER_OF_CORES                               (N > 1)
-    #define configUSE_TRACE_FACILITY                         1
-    #define configUSE_STATS_FORMATTING_FUNCTIONS             1
-
-Coverage for: 
-        void vTaskList( char * pcWriteBuffer )
-        if( pxTaskStatusArray != NULL ) = False
-*/
-void test_v_task_list_case_no_task_created( void )
+/**
+ * @brief vTaskList - No task is created.
+ *
+ * This API is called before any task is created.
+ *
+ * <b>Coverage</b>
+ * @code{c}
+ * if( pxTaskStatusArray != NULL )
+ * {
+ *     ...
+ * }
+ * @endcode
+ * ( pxTaskStatusArray != NULL ) is false.
+ */
+void test_coverage_vTaskList_no_task_created( void )
 {
-    static char	buff[ 800 ] = { 0 };
- 
-    //Call the List
-    vTaskList(buff);
+    char pcWriteBuffer[ TEST_VTASKLIST_BUFFER_SIZE ] = "Test"; /* Validate the string is overwritten in the API call. */
 
-}
-/*
-The kernel will be configured as follows:
-    #define configNUMBER_OF_CORES                               (N > 1)
-    #define configUSE_TRACE_FACILITY                         1
-    #define configUSE_STATS_FORMATTING_FUNCTIONS             1
+    /* Setup the variables and structure. */
+    uxCurrentNumberOfTasks = 0;         /* No task is created. */
+    UnityMalloc_StartTest();
 
-Coverage for: 
-        void vTaskList( char * pcWriteBuffer )
-        case eDeleted
-*/
-void test_v_task_list_case_eDeleted( void )
-{
-    static char	buff[ 800 ] = { 0 };
+    /* API calls. */
+    vTaskList( pcWriteBuffer );
 
-    TaskHandle_t xTaskHandles[configNUMBER_OF_CORES+1] = { NULL };
+    /* Validation. */
+    /* Verify the malloc allocate count. */
+    UnityMalloc_EndTest();
+    /* No task is created. A string with zero legnth is returned. */
+    TEST_ASSERT_EQUAL( pcWriteBuffer[ 0 ], 0x00 );
```

Comment:
these are in the reverse order. 
TEST_ASSERT_XXX( expected_value, returned_value_to_test);

Reply:
Thanks for the comment. Update in new commit.